features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/transport/Netflow9MessageBuilder.java

```diff
@@ -78,7 +79,7 @@ public FlowMessage.Builder buildMessage(final Iterable<Value<?>> values, final R
         Long dstVlan = null;
         Long flowActiveTimeout = this.flowActiveTimeoutFallback;
         Long flowInActiveTimeout = this.flowInactiveTimeoutFallback;
-        Long sysUpTime = null;
+        Long sysUpTime = 0L;
```

Comment:
My nit, why not keep this as `null` and then change `line 260` below to read as:

`        long bootTime = sysUpTime != null ? timeStampInMsecs - sysUpTime : timeStampInMsecs;`

Reply:
that works too, thanks